examples/wc_cn/README.txt

```diff
@@ -0,0 +1,19 @@
+License
+
+Call to Arms (Lu Xun), collection of short stories.
+
+LuXun_black : Thousand pointing fingers, willing ox bow (Zhao Yannian's Woodcut 1961)
+
+LuXun_color : Thousand pointing fingers, willing ox bow (Yin Shoushi's Chinese Ink 1973)(after image processing)
+
+
+According to the copyright law of People's Republic of China, Call to Arms and two works of Artcan be used as a scientific research and personal study, a small amount of copying and translation.
+
+But if you want to use this literary work in other ways, please refer to the copyright law of the People's Republic of China or the relevant laws of the corresponding country.
+
+P.S. Mr. Lu Xun is a spiritual mentor of my life and a good friend of mine. I like him very much.
```

Comment:
Whose voice is this here?

Reply:
......I like Lu Xun .but now,I don't think it really fits in here.I'll fix this mistake.